### ISSUE TITLE
No image color verification

### ISSUE THREAD

**Issue by cola6969** [drov0/python-imagesearch#16]

**Comment by drov0** [comment]
Indeed, images are converted to grey in order to speed up search. I don't have the time right now but when I'll do maintenance I'll add the option to check colors

**Comment by cola6969** [comment]
@drov0  Ok ;C but can we do this with color verification? Because in my test i have two dots with two different colors.

**Comment by KacperGwozdz** [comment]
@drov0 Verification with colors can solve many of our project's cases, so it will be very helpful for us. We are looking forward when it will be ready! :)

**Comment by NoPantsPirate** [comment]
Came here looking for this issue as I am also experiencing it. Basically if you have a button with a default disabled state, (which is typically gray), and is a different color when enabled/active, the library will detect the disabled button right away but what we are searching for is the colorful active button. I would imagine this must be a common scenario to encounter

**Comment by NoPantsPirate** [comment]
So I've gotten around this by using the included `pyautogui.locate()` method. I've set an optional parameter for `color` to make use of it instead of the existing route. I'll tidy it up and consider a PR

**Comment by drov0** [comment]
Cool ! that would be nice
